src/main/java/practice/pankaj/Program3.java

```diff
@@ -0,0 +1,29 @@
+package Pankaj;
+import java.util.*;
+
+public class Program3 {
+
+
+    public static void main(String[] args)
+    {
+        int n, max;
+        Scanner s = new Scanner(System.in);
+        System.out.print("Enter number of elements in the array:");
+        n = s.nextInt();
+        int a[] = new int[n];
+        System.out.println("Enter elements of array:");
+        for(int i = 0; i < n; i++)
+        {
+            a[i] = s.nextInt();
+        }
+        max = a[0];
+        for(int i = 0; i < n; i++)
+        {
+            if(max < a[i])
+            {
+                max = a[i];
+            }
+        }
+        System.out.println("Maximum value:"+max);
+    }
+}
```

Comment:
public static void main(String[] args)
    {
        int num[] = {2,5,6,7,2,3,4,15,36,963,54,631,89};
        System.out.print("\nMethod 1\nNumbers before sorting \t ");
        for (int i=0;i<num.length;i++)
        {
            System.out.print(" " + num[i]);
        }
        Arrays.sort(num);
        System.out.print("\nNumbers After sorting \t ");
        for (int i=0;i<num.length;i++)
        {
            System.out.print(" " + num[i]);
        }
        int j,k;
        j = num.length-1;
        k=0;
        System.out.println("\nThe largest number from the Array:\t"+num[j]);
        System.out.println("The Smallest number from the Array:\t"+num[k]);
    }

Reply:
you can also use this method